src/lib/challenge.ts

```diff
@@ -48,3 +52,26 @@ export async function generate (size: number = 32): Promise<string> {
     throw error
   }
 }
+
```

Comment:
there is a need to put into system configuration what type of publicKey is used (RSA or EC with additional parameters), also which hash algorithm is used to sign and verify SHA256 - and what else is allowed by fido spec? 
Let don't hardcode these parameters in the code

Reply:
@eoln The verifier works with many different types of keys without an explicit need to specify which keys it should work with. FIDO allows RSA and EC for some curves. I have tested the function for RSA and the EC I believe we are using in the PISP app.

Reply:
@jgeewax there is a need for a designer decision: what type of hash/verify algorithm should we implement?

Reply:
@eoln we are accepting many types of keys and using SHA256 for hashing. I have confirmed with with @jgeewax . I could add more documentation on which keys are accepted and which have been tested properly.

Reply:
That's great we will support many types of keys and hashing algorithms! 
@jgeewax How the parties will get an agreement which type of keys & algorithms will be used to sing and verify for the given challenge? Maybe the key/hash type should be the part of requests so mobile app will know how to sign, and dfsp/auth-service will know how to verify signature. Then it means that key and hash algorithm types shouldn't be hardcoded in the code but passed via parameters

Reply:
Just putting in writing what we discussed in stand-up before: the different types of keys should be specified in the key content itself. The choice of what type of key to use (RSA, EC, etc) is up to the device. For now I say we just say we'll use RSA for everything and make sure that works.